Kernel/Syscall.cpp

```diff
@@ -161,6 +168,7 @@ NEVER_INLINE void syscall_handler(TrapFrame* trap)
 
     current_thread->yield_if_stopped();
 
+#if ARCH(X86_64)
```

Comment:
There should be a TODO_AARCH64 here for this security mitigation.

Reply:
I added a FIXME, instead of inserting a `TODO_AARCH64` as that will cause a crash at runtime, and this code works fine without the security mechanisms in place.